lib/async_hooks.js

```diff
@@ -60,19 +60,11 @@ async_wrap.setupHooks({ init,
                         destroy: emitDestroyN });
 
 // Used to fatally abort the process if a callback throws.
-function fatalError(e) {
```

Comment:
If anything I'd like to export the code in `Environment::AsyncHooks::pop_ids()` and use that as the uniform way to exit the process.